src/Compilers/CSharp/Portable/Emitter/EditAndContinue/CSharpSymbolMatcher.cs

```diff
@@ -112,9 +110,9 @@ private Cci.IDefinition VisitDefInternal(Cci.IDefinition def)
                     }
 
                     var nestedType = type.AsNestedTypeDefinition(_sourceContext);
-                    Debug.Assert(nestedType != null);
+                    RoslynDebug.AssertNotNull(nestedType);
```

Comment:
This change is usually not necessary if we have a netcoreapp3.1 or greater target in the project.